sample/src/main/java/co/nimblehq/sample/ui/login/LoginViewModel.kt

```diff
@@ -18,38 +14,8 @@ class LoginViewModel(private val loginRepository: LoginRepository) : ViewModel()
 
     fun login(username: String, password: String) {
         // can be launched in a separate asynchronous job
-        val result = loginRepository.login(username, password)
-
-        if (result is Result.Success) {
-            _loginResult.value =
-                LoginResult(success = LoggedInUserView(displayName = result.data.displayName))
-        } else {
-            _loginResult.value = LoginResult(error = R.string.login_failed)
-        }
-    }
-
-    fun loginDataChanged(username: String, password: String) {
-        if (!isUserNameValid(username)) {
-            _loginForm.value = LoginFormState(usernameError = R.string.invalid_username)
-        } else if (!isPasswordValid(password)) {
-            _loginForm.value = LoginFormState(passwordError = R.string.invalid_password)
-        } else {
-            _loginForm.value = LoginFormState(isDataValid = true)
-        }
+        _loginResult.value =
+            LoginResult(success = LoggedInUserView("test", "token"))
```

Comment:
what do `test` and `token` mean here? 🤔

Reply:
@luongvo Same above, I suggest we should not focus on sample at this time. We will check on final PR.

Reply:
@doannimble With the inputs for multiple fields, I'd recommend adding the field name before the value. Ex: LoggedInUserView(displayName = "test", token = "token"))

Reply:
@hoangnguyen92dn Yes, it should be like that.